client/src/containers/interventions/new/component.tsx

```diff
@@ -17,33 +14,31 @@ import Step2 from './step2';
 
 const STEP1: Step = {
   id: 'Step 1',
-  slug: 'intervention_step1',
+  slug: 1,
```

Comment:
```suggestion
  slug: 'step-1',
```
following my comment above about the slugs, if you want to keep a index, use the `id` instead